.github/workflows/spell-checker.yml

```
@@ -0,0 +1,32 @@
+name: Spell Checkerr
+on:
+  push:
+    branches:
+      - 'master'
+      - '[0-9]+.[1-9][0-9]*.x'
+    paths:
+      - 'docs/**'
+  pull_request:
+    branches:
+      - 'master'
+      - '[0-9]+.[1-9][0-9]*.x'
```

We also need to run this while target PRs against the `page` branch
```suggestion
      - '[0-9]+.[1-9][0-9]*.x'
      - page
```